CHANGELOG.md

```diff
@@ -609,6 +609,7 @@ Released with 1.0.0-beta.37 code base.
 
 -  Fixed types for getPastEvents (#4955) (#5260)
 -  Fix Log type by adding missing `removed` property (#4877)
+-  Fixed types for `web3.utils._jsonInterfaceMethodToString` (#5550)
```

Comment:
Kindly rebase your branch and put this under unreleased `1.8.2` . Thanks

Reply:
Done.

Should I squash commits now or during the PR merge?

Reply:
squash commits will happen during pr merge